tests/installation/zdup_prepare_repos.pm

```diff
@@ -0,0 +1,102 @@
+# SUSE's openQA tests
```

Comment:
I think the splitout of this test module could be a first PR which we can merge more easily.